perun-ldapc-ada/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/GroupEventProcessor.java

```diff
@@ -36,6 +36,18 @@ public void processMemberAdded(String msg, MessageBeans beans) {
 		}
 	}
 
+	public void processMemberRemoved(String msg, MessageBeans beans) {
+		if(beans.getGroup() == null || beans.getMember() == null) {
+			return;
+		}
+		try {
+			log.debug("Removing member {} from group {}", beans.getMember(), beans.getGroup());
+			perunGroup.removeMemberFromGroup(beans.getMember(), beans.getGroup());
+		} catch (NamingException | InternalErrorException e) {
+			log.error("Error removing member {} from group {}: {}", beans.getMember().getId(), beans.getGroup().getId(), e.getMessage());;
+		}
+	}
+
```

Comment:
This is the first time you need method to remove member from group? So in the basic implementation you are not removing members from group or there is different logic in it? What about "remove Member from Group totally" message?

Reply:
Yes to the first two, "totally" was added too.